src/operations/secrets.md

```diff
@@ -0,0 +1,83 @@
+# Managing Secrets
+
+The purpose of this document is to provide information to contributors on how to store and view the Toronto Community Network's "secrets", such as passwords, keys, credentials, and other information kept private for security reasons.
+
+## Introduction
+The Toronto Community Network uses [Bitwarden](https://bitwarden.com/) to share and manage secrets across the organization. Bitwarden is a free and open-source password management service that stores sensitive information (such as website credentials) in an encrypted vault.
+
+Bitwarden supports the following clients:
+
+- Desktop (Linux/MacOS/Windows)
+- Web Browser (Chrome/Safari/Firefox/Vivaldi/Opera/Brave/Edge/Tor Browser)
+- Mobile (Android/iPhone)
+- Command Line (via NPM)
+
+For more information about supported clients, see Bitwarden's [download options](https://bitwarden.com/download/).
+
+
+## Accessing Secrets
+The Toronto Community Network is using the self-hosted version of Bitwarden, and can be accessed at [https://tomesh.infectedroot.com](https://tomesh.infectedroot.com).
```

Comment:
how much trouble would it be to change the hostname to a tomesh.net one in bitwarden? We can cname it for now.